### ISSUE TITLE
Quest version? 

### ISSUE THREAD

**Issue by kiarfuzzy** [Loloppe/BeatSaber_BeatmapScanner#2]
Not really an issue, but do you have any plans to make it for quest 2 too?

**Comment by Loloppe** [comment]
Not really, I don't have a Quest 2 and this is probably a thousand time more complicated to make/port on quest because of all the dependencies and stuff (mostly UI). If someone really want something like this on Quest 2, their best bet is to commission a Q2 modder on the commission discord. I would guess it might cost a few hundred $ to get a working version or something idk.

**Comment by kiarfuzzy** [comment]
Sadto hear, hopefully someone will pick it up and port it, because it s a pretty cool plugin
